src/controllers/authContoller.ts

```diff
@@ -424,7 +435,7 @@ export const updateProfile = async (
     return res.status(400).json({ errors: errors.array() });
   }
 
-  const { name, phoneNumber, gender } = req.body;
+  const { name, phoneNumber, gender, batch } = req.body;
```

Comment:
```   
const userBatch = await prisma.batch.findUnique({
      where: {
        batch: batch.toString()
      }
    });

```

where is this check, we can have dropdown in frontend ?

Reply:
we already have it in frontend.

Reply:
browser is not only the client will be expecting :)